bolt/database.rules.bolt

```diff
@@ -6,9 +6,12 @@ path /events {
     write() {
       isAuthed() &&
       (
-        isSuperAdmin() ||
-        this !== null && prior(this) === null ||
-        auth.uid === prior(root.events[eventId].region)
```

Comment:
Removing the editability by region restricts events to just the ones you created. If you're unavailable and the event needs to be changed, another mentor in your city should be able to edit your event. All users are [assigned a region](https://github.com/askadev/askadev.org/blob/master/bolt/database.types.bolt#L38) when their [username is added](https://github.com/askadev/askadev.org/blob/master/scripts/components/Admin/AddUsername/Presenter.js#L59-L62).

Reply:
I must have an old version of the DB I couldn't find where they were assigned.